controllers/tasks.js

```diff
@@ -144,10 +153,24 @@ const updateTaskStatus = async (req, res) => {
     const { id: userId } = req.userData;
     const task = await tasks.fetchSelfTask(taskId, userId);
 
-    if (task.taskNotFound) return res.boom.notFound("Task doesn't exist");
-    if (task.notAssignedToYou) return res.boom.forbidden("This task is not assigned to you");
-    if (task.taskData.status === "VERIFIED")
+    if (task.taskNotFound) {
+      logger.error(`Error while updating taskId ${req.params.id}: Invalid taskId passed`);
+      return res.boom.notFound("Task doesn't exist");
+    }
+    if (task.notAssignedToYou) {
+      logger.error(`Error while updating taskId ${req.params.id}: Forbidden user`);
+      return res.boom.forbidden("This task is not assigned to you");
+    }
+    if (task.taskData.status === "VERIFIED") {
```

Comment:
Let's have this from a constants file please

Reply:
Thanks for pointing it out. Refactored all the places where task status was not imported from constants file